### PR TITLE
change license header format to something jsdoc friendly

### PR DESCRIPTION
**Pull request by paul-marechal**
#### What it does

This is a silly problem but the earlier we fix it the "smaller" the PR is (currently ~~1619~~ 1625 files).

The way JSDoc works, if you have a comment block above a symbol declaration then both will be assumed to go together.

See the problem: https://eclipse-theia.github.io/theia/docs/next/modules/plugin_ext.html#pluginPathsServicePath

image

Note that we shouldn't see our license in this place, but it happens because we have the following:

```ts
/********************************************************************************
 * Copyright (C) 2018 Red Hat, Inc. and others.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * This Source Code may also be made available under the following Secondary
 * Licenses when the conditions for such availability set forth in the Eclipse
 * Public License v. 2.0 are satisfied: GNU General Public License, version 2
 * with the GNU Classpath Exception which is available at
 * https://www.gnu.org/software/classpath/license.html.
 *
 * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 ********************************************************************************/

export const pluginPathsServicePath = '/services/plugin-paths';
```

In the above example the whole license header gets "attached" to the unrelated const.

The only way I found to fix this is to just change the way we write our license header:

```ts
// *****************************************************************************
// Copyright (C) 2022 COMPANY and others.
//
// This program and the accompanying materials are made available under the
// terms of the Eclipse Public License v. 2.0 which is available at
// http://www.eclipse.org/legal/epl-2.0.
//
// This Source Code may also be made available under the following Secondary
// Licenses when the conditions for such availability set forth in the Eclipse
// Public License v. 2.0 are satisfied: GNU General Public License, version 2
// with the GNU Classpath Exception which is available at
// https://www.gnu.org/software/classpath/license.html.
//
// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
// *****************************************************************************
```

This PR is split in 2 commits to separate the script I used to do the changes from running said script.

#### How to test

Everything should work like before.

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by vince-fugnitto — APPROVED** [review]
The changes look good to me :+1: 
I confirmed that:
- the new snippet works correctly
- the changes result in better automated documentation:

_left_ is the pull-request:

image

image

**Review by JonasHelming — APPROVED** [review]
Looks good to me, I only did a high level check, though

**Comment by paul-marechal** [comment]
Re-based the PR and re-generated the headers for the new files. Will wait for CI to turn green before merging :)

**Pull request merged**
